src/components/Tooltip/Tooltip.tsx

```diff
@@ -72,36 +72,40 @@ export enum TooltipAlignment {
     End = "End",
 }
 
-const placementMap: Record<string, VariationPlacement> = {
-    ["top-Start"]: "top-start",
-    ["top-End"]: "top-end",
-    ["bottom-Start"]: "bottom-start",
-    ["bottom-End"]: "bottom-end",
-    ["left-Start"]: "left-start",
-    ["left-End"]: "left-end",
-    ["right-Start"]: "right-start",
-    ["right-End"]: "right-end",
+const placementMap: Record<`${TooltipPosition}-${TooltipAlignment}`, Placement> = {
+    ["Top-Start"]: "top-start",
```

Comment:
Can be written "[`${TooltipPosition.Top}-${TooltipAlignment.Start}`]"

Reply:
Typescript didn't like that at all.